docs/source/api/scene.rst

```diff
@@ -67,6 +75,15 @@ scene
       :returns: The scene's sun entity (if it exists)
       :rtype: entity
 
+   .. lua:attribute:: canvas : ui.canvas
+
+         :returns: The main ui canvas of the scene (ui starts here)
+         :rtype: ui.canvas
```

Comment:
Similar to the previous comment, I think these lines may be indented one tab too far

On the returns description you can probably capitalise `UI`:
`:returns: The main UI canvas of the scene`